libvcx/src/agency_comm/util.rs

```diff
@@ -0,0 +1,9 @@
+use agency_comm::agency_settings;
+use error::VcxResult;
+use utils;
+
+//Todo: change this RC to a u32
```

Comment:
There is no rc in here, also, I would name this file `utils.rs`, or maybe `utils/comm.rs` (and move `agent_utils` to `utils/agent.rs`)